.github/workflows/jekyll-gh-pages.yml

```diff
@@ -0,0 +1,57 @@
+# Sample workflow for building and deploying a Jekyll site to GitHub Pages
+name: Deploy Jekyll with GitHub Pages dependencies preinstalled
+
+on:
+  # Runs on pushes targeting the default branch
+  push:
+    branches: ["use-jekyll"]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# Sets permissions of the GITHUB_TOKEN to allow deployment to GitHub Pages
+permissions:
+  contents: read
+  pages: write
+  id-token: write
+
+# Allow only one concurrent deployment, skipping runs queued between the run in-progress and latest queued.
+# However, do NOT cancel in-progress runs as we want to allow these production deployments to complete.
+concurrency:
+  group: "pages"
+  cancel-in-progress: false
+
+jobs:
+  # Build job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup Pages
+        uses: actions/configure-pages@v3
+      - name: Download Bootstrap 5 dependencies
+        run: |
+          wget "https://github.com/twbs/bootstrap/archive/v5.3.2.zip" && \
+          unzip v5.3.2.zip -d ./_sass && mkdir ./_sass/bootstrap && \
+          mv ./_sass/bootstrap-5.3.2/scss ./_sass/bootstrap/scss && \
+          rm v5.3.2.zip && rm -r ./_sass/bootstrap-5.3.2
```

Comment:
Seems a bit sad to do this on every build but I'm not really convinced about the merits of maintaining our own image (if thats even possible). 

Separately but relatedly do we have to depend on 5.3.2? Is there a way for us to depend on 5.3.latest or even 5.latest ?

Reply:
> Separately but relatedly do we have to depend on 5.3.2? Is there a way for us to depend on 5.3.latest or even 5.latest ?

I couldn't find a way to do so with the Sass sources, except maybe for downloading a zip of the latest (and potentially unstable) main branch sources in the repo, instead of the zip of the (stable) v5.3.2 tag sources which we download currently.

Everything is easier when you go via CDN, except you can't get the Sass sources (which have all the functions for inserting your own brand colours) over CDN.

Reply:
Thanks for checking.